server.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/acm-uiuc/arbor/server"
+	"github.com/acm-uiuc/groot-api-gateway/config"
```

Comment:
What's the difference between groot-api-gateway and arbor? They both map to the arbor repo